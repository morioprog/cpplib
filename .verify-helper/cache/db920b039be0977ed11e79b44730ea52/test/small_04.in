5 20
807837051411 648609035052 401038253266 -36998149630 251566274515
0 1 2 -399047092549
1 3 5 919047250663
0 1 3 299217277322
1 1 4 -219872072499
3 2 5
0 2 4 -998965408711
2 1 2 12620992124
3 3 5
2 4 5 30179096460
0 0 1 751849001040
2 3 4 44880874850
3 1 5
1 0 4 581398674451
1 1 4 -793311291492
1 2 4 555081088841
0 0 2 -198789196440
2 1 4 -28741067519
0 0 4 862044696308
2 0 3 -237237428000
1 1 4 167649678777
