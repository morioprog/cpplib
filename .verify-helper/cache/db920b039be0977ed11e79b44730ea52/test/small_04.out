2137311778648
-79918158048
-1211074675824
